.gitignore

```diff
@@ -1 +1,2 @@
-/vendor/
\ No newline at end of file
+/vendor/
+composer.lock
```

Comment:
Could you please undo this unrelated change?

Reply:
You have more problems in this case with composer.lock than it helps.

example:
composer.lock php-pm last commit: cd12ed86ac4d043d584cb1717a2bae64b576c1a0
php-pm last commit is 70e7b3134e85b30489c3c9e1a670e1ced83afae5

And when you the README.md take seriously it will not work.